### PR TITLE
Fix the three signet staffs to match retail-well, more than before...

### PR DESCRIPTION
**Pull request by helixhamin**
Restrict to player using and target being of the proper nation, and removal of Sanction and Sigil.

### CODE REVIEW

**Comment by helixhamin** [comment]
https://github.com/DarkstarProject/darkstar/issues/1491

File: scripts/globals/items/federation_signet_staff.lua [hunk 1]
```diff
@@ -19,6 +19,12 @@ end;
 -----------------------------------------
 
 function onItemUse(target)
-    target:delStatusEffect(EFFECT_SIGNET);
-    target:addStatusEffect(EFFECT_SIGNET,0,0,18000);
+    if (player:getNation() == WINDURST)
```
Comment (xdemolish) [inline scripts/globals/items/federation_signet_staff.lua]: did you test?

Comment (helixhamin) [inline scripts/globals/items/federation_signet_staff.lua]: Need to wait until I get home. At work right now. Will report back once I have.

**Comment by helixhamin** [comment]
@xdemolish Yeah, you are right, it fails. Completely. I can get it to remove sigil/sanction/signet, and apply a new signet, but only by removing the if statements. And it will not buff others, just the person using, so not sure it is any better than before.
The logic is lost on me on how to apply the restrictions. I should just close.

**Pull request closed without merging**